heron/api/src/java/BUILD

```diff
@@ -11,23 +12,37 @@ api_deps_files =  \
         "//third_party/java:kryo",
     ]
 
+# Kryo dependency should be specified in POM
+api_deps_files_nl =  \
```

Comment:
also, the nl part of the name isn't informative. I assume it relates to neverlink, but could we instead name based on the functional difference, like something to do with excluding from the artifact? The casual reader won't know what nverlink means.